src/cargo/core/features.rs

```diff
@@ -400,6 +400,36 @@ impl FromStr for FixEdition {
     }
 }
 
+/// The value for `-Zdetect-antivirus`.
+#[derive(Debug, Deserialize, Default, PartialEq, Eq, Hash, Copy, Clone)]
+#[serde(rename = "lowercase")]
+pub enum DetectAntivirus {
+    /// Always detect antivirus.
+    ///
+    /// This is useful when testing the feature, but isn't expected to be
+    /// useful to the general user.
+    Always,
+    /// Detect antivirus, but only when deemed reasonable.
+    ///
+    /// This is intended to be the default in the future.
+    Auto,
+    /// Never attempt to detect antivirus.
+    #[default]
+    Never,
+}
+
+impl FromStr for DetectAntivirus {
+    type Err = anyhow::Error;
+    fn from_str(s: &str) -> Result<Self, <Self as FromStr>::Err> {
+        Ok(match s {
+            "always" => Self::Always,
+            "auto" => Self::Auto,
+            "never" => Self::Never,
+            _ => bail!("invalid `-Zfix-edition, expected `always`, `auto` or `never`, got `{s}`"),
```

Comment:
The `-Zfix-edition` is wrong here. Probably best to just remove that part of the message.